### DISCUSSION TITLE
SbieDll.dll failure

### DISCUSSION THREAD

**Discussion by GLoren1** [sandboxie-plus/Sandboxie#1619] [Q&A Feedback]
When I try to run certain MSI files to install programs in sandbox, I get the error message:

"Initialization of the dynamic link library "C:\Program Files\Sandboxie-Plus\SbieDll.dll failed. The process is terminating abnormally.". 

 I've uninstalled and re-installed Sandboxie Plus, but it hasn't solved the problem.

Any help would be appreciated

**Comment by isaak654** [comment]
Reposted in issue https://github.com/sandboxie-plus/Sandboxie/issues/1620

**Reply by DavidXanatos** [reply]
thats strange msi files should not install a sbiedll, can you point me to a download to soem of the problematic msi's please: https://github.com/sandboxie-plus/Sandboxie/issues/1620
